gemrb/core/GUI/GameControl.cpp

```diff
@@ -325,11 +325,7 @@ void GameControl::CreateMovement(Actor *actor, const Point &p, bool append, bool
 // can we handle it (no movement impairments)?
 bool GameControl::CanRun(const Actor *actor) const
 {
-	if (!actor) return false;
-	if (actor->GetEncumbranceFactor(true) != 1) {
-		return false;
-	}
-	return true;
+	return (actor && actor->GetEncumbranceFactor(true) != 1);
 }
```

Comment:
This chunk has nothing to do with the bug or its fix, please remove it. The golden rule is to do one thematic thing per commit, so it's easier to review, rereview and test when bisecting.

Reply:
tho do feel free to add it as an _additional_ commit

Reply:
ok